tsconfig-strict.json

```diff
@@ -368,6 +368,28 @@
     "core/templates/pages/exploration-editor-page/feedback-tab/services/thread-status-display.service.spec.ts",
     "core/templates/pages/exploration-editor-page/history-tab/services/version-tree.service.ts",
     "core/templates/pages/exploration-editor-page/history-tab/services/version-tree.service.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/confirm-discard-changes-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/confirm-discard-changes-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/confirm-leave-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/confirm-leave-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/editor-reloading-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/editor-reloading-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/exploration-publish-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/exploration-publish-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/help-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/help-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/lost-changes-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/lost-changes-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/post-publish-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/post-publish-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/save-validation-fail-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/save-validation-fail-modal.component.spec.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/save-version-mismatch-modal.component.ts",
+    "core/templates/pages/exploration-editor-page/modal-templates/save-version-mismatch-modal.component.spec.ts",
```

Comment:
Why do you remove these files up above?

Reply:
I just deleted the angularJS file(.controller), as these files are already migrated to the component.